templates/cisco_xr_show_ipv6_neighbors.template

```diff
@@ -0,0 +1,18 @@
+Value ADDRESS (\S+)
+Value AGE (\S+)
+Value MAC (\S+)
+Value STATE (\S+)
+Value INTERFACE (\S+)
+Value LOCATION (\S+)
+
+Start
+  ^\S+\s+\S+\s+\S+\s+\S+:\S+:\S+\.\S+\s+\S+\s*
+  ^IPv6\s+Address\s+Age\s+Link-layer\s+Add(r?)\s+State\s+Interface\s+Location\s*$$ -> Begin
+  ^\s*$$
+  ^. -> Error
+
+Begin
+  ^${ADDRESS}\s\s+${AGE}\s+${MAC}\s+${STATE}\s+${INTERFACE}\s+${LOCATION}\s*$$ -> Record
```

Comment:
I would say that we should capture Mcast entries as well. Add the following above L15

`  ^\[${ADDRESS}\]\s+${AGE}\s+${MAC}\s+${STATE}\s+${INTERFACE}\s+${LOCATION}\s*$$ -> Record`

Reply:
Sorry you can just update the existing line to:
`^\[?${ADDRESS}\]?\s+${AGE}\s+${MAC}\s+${STATE}\s+${INTERFACE}\s+${LOCATION}\s*$$ -> Record`

Reply:
I assume it would make  sense to remove line 16  as well, since that gets captured by line 15 now?

    ^\s*\[Mcast\s+adjacency\]